menderqa_pipeline.go

```diff
@@ -491,6 +491,10 @@ func getClientBuildParameters(
 		// that we can at least build it. MEN-6116 exists to clean this
 		// up later.
 		qemuParam = "false"
+		buildParameters = append(
+			buildParameters,
+			&gitlab.PipelineVariable{Key: "BUILD_CPP_CLIENT", Value: "true"},
```

Comment:
Is it not possible to determine this from the branch or version? I suppose that with a `meta-mender/feature-c++-client` branch, we can default to always build the C++ version, and then we shouldn't need this, or?

Reply:
I thought the plan was to merge the `meta-mender` branch to `master`, and not rely on a feature branch?

Reply:
Hmm. Honestly I don't know which is better, I can see pros and cons with both approaches. Maybe having a branch is a bit safer though? To not stir up too much trouble while we're still working on this?

Reply:
Ref: f2f discussion

We will go down this route for now.